pkgs/top-level/aliases.nix

```diff
@@ -333,6 +333,7 @@ mapAliases ({
   ppl-address-book = throw "deprecated in 2019-05-02: abandoned by upstream.";
   processing3 = processing; # added 2019-08-16
   procps-ng = procps; # added 2018-06-08
+  pygmentex = texlive.bin.pygmentex; # added 2019-12-15
```

Comment:
This does not cover the use case introduced in #56063. Is that ok?

Reply:
It is ok.

What matters is the availability of both the latex package and the python script when compiling a document that uses `pygmentex`. Until now they were not provided by the same package.

Reply:
Well, I believe this PR will improve user experience by providing a working python script. However I also care about backward compatibility. The workaround from #10885 is old, simple and expected to be widely used, and the good news is that it should not break with this alias. Supporting the workaround suggested in #56063 would require adding the legacy stuff to `texlive.bin.pygmentex`, which I would like to avoid.